test/e2e/jenkins.go

```diff
@@ -51,12 +51,39 @@ func getJenkinsMasterPod(t *testing.T, jenkins *v1alpha2.Jenkins) *corev1.Pod {
 	return &podList.Items[0]
 }
 
+func getServiceAccount(t *testing.T, jenkins *v1alpha2.Jenkins) *corev1.ServiceAccount {
+	saName := fmt.Sprintf("jenkins-operator-%s", jenkins.Name)
+	sa, err := framework.Global.KubeClient.CoreV1().ServiceAccounts(jenkins.ObjectMeta.Namespace).Get(saName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	return sa
+}
+
+type JenkinsE2EConfiguration struct {
+	name              string
+	namespace         string
+	priorityClassName string
+	plugins           []v1alpha2.Plugin
+	seedJob           *[]v1alpha2.SeedJob
+	groovyScripts     v1alpha2.GroovyScripts
+	casc              v1alpha2.ConfigurationAsCode
+}
+
+func createJenkinsCRFromSample(t *testing.T, sample JenkinsE2EConfiguration) *v1alpha2.Jenkins {
+	j := createJenkinsCR(t, sample.name, sample.namespace, sample.seedJob, sample.groovyScripts, sample.casc, sample.priorityClassName)
+	plugins := sample.plugins
+	if len(plugins) > 0 {
+		j.Spec.Master.Plugins = plugins
+	}
```

Comment:
it won't work because Jenkins CR is already saved without pulgins

Reply:
fixed by doing the assignment before